src/security_utilities_rust/src/microsoft_security_utilities_core/identifiable_scans.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-use std::rc::Rc;
+use std::sync::Arc;
```

Comment:
Can we just remove these fully? I believe since the validator is just fn you can do it, right?

Reply:
We could, but then it also prevents anyone else from implementing validators that are not compile-time constant. I guess we could always loosen the constraint later in the case that it turns out that we wish to support that.

Reply:
Interesting, good point. @michaelcfanning likely owns that decision.

Reply:
Will do! Will also add a changelog entry

Reply:
Have left it as-is for now, with a changelog entry. I'm in EU timezones, so will not be available for ~10 hours now.

Reply:
I think in general it's fine for this library to be in a bit of flux. A tighter constraint that we loosen later is easy to handle from a compat standpoint than trying to move the other direction.

If Beau doesn't have a strong feeling, let's get this in and we can revisit as Johannes points out later if it comes up.